node, npx: fix of a command and clearer descriptions (#8899)
tldr-pages/tldr: pages/common/npx.md

> Execute binaries from `npm` packages.
> More information: <https://github.com/npm/npx>.

- Execute the command from a local or remote `npm` package:

> `npx {{command}} {{command_arguments}}`

Reviewer: This should be the package name, e.g. `react-native-cli`, not the executable `react-native`
```suggestion
`npx {{package}} {{command_arguments}}`
```

Author: Actually there's good overlapping between the two since, most of the time, commands have the same name of packages.
Because of that is both possible to call `npx react-native-cli` or `npx react-native`.

I could suggest to use a syntax that can include both as sometimes is shown in the man pages: ```npx {{package|command}} {{command_arguments}}```

What do you think about that?

Author: After all the result of `man npx` is
```
[....]

 npx <pkg>[@<specifier>] [args...]
 npx -p <pkg>[@<specifier>] <cmd> [args...]
 npx -c '<cmd> [args...]'
 npx -p <pkg>[@<specifier>] -c '<cmd> [args...]'

[...]        ```

Reviewer 2: @ffranchina `{{package|command}}` sounds good to me!

> `npx {{command}} {{command_arguments}}`

Reviewer 3: ```suggestion
`npx {{command}} {{arg1 arg2 ...}}`
```

Mandatory fix: [ellipsis must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).

> - In case multiple commands with the same name exist, it is possibile to specify the package name:

Reviewer: ```suggestion
- In case multiple commands with the same name exist, it is possible to specify the package name:
```

`npx --package {{package_name}} {{command}}`

- Run a command if it exists in the current path or in `node_modules/.bin`:

`npx --no-install {{command}} {{command_arguments}}`

> - Execute the command suppressing any output from `npx` itself:

Reviewer 3: ```suggestion
- Execute a specific command suppressing any output from `npx` itself:
```
Optional fix: `specific` may sound better.

> - Execute the command suppressing any output from `npx` itself:

Reviewer: ```suggestion
- Execute the command from a given npm package suppressing any output from `npx` itself:
```

> `npx --quiet {{command}} {{command_arguments}}`

Reviewer 3: ```suggestion
`npx --quiet {{command}} {{arg1 arg2 ...}}`
```
Mandatory fix: [ellipsis must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).

`npx {{module_name}} {{command_arguments}}`

> - In case a module has multiple binaries, specify the package name:

Reviewer: I don't like this example description. what is module vs package?

Author: Actually this string it a bit confusing. I made a bit of research about the `npx` jargon and, in the original man page there's:
`npx [options] [-p|--package <pkg>]... <command> [command-arg]...`
Therefore a _package_/_module_ is a JavaScript distributable collection of source code. I will apply some changes keeping this into account.
If the changes will be accepted, I will modify also https://github.com/tldr-pages/tldr/pull/8898
